tasks/9-img-scan.yaml

```diff
@@ -89,7 +89,7 @@ spec:
           PATH_TO_IMAGE="/var/oci/image"
           echo -e "Trivy Security Scan image in registry"
           trivy image --exit-code 0 --input ${PATH_TO_IMAGE}
-          trivy image --exit-code 1 --severity CRITICAL --input ${PATH_TO_IMAGE}
+          trivy image --exit-code 1 --severity CRITICAL --ignore-unfixed --input ${PATH_TO_IMAGE}
```

Comment:
Can you make this a parameter to the task, and make it default to this. But this ability if someone wants to use the task and doesn't want the flag to remove the flag. 

I'm thinking a param like "ignoreUnfixed" default to "true", if true then add the flag. if user pass "false" then don't use the flag

Reply:
Implemented for both trivy scan tasks using a command-specific environment variable set from the task parameter.